Dockerfile

```diff
@@ -7,5 +7,5 @@ ADD python /cuML/python
 ADD setup.py /cuML/setup.py
 
 WORKDIR /cuML
-RUN source activate gdf && conda install -c pytorch cython faiss-gpu
-RUN source activate gdf && python setup.py install
+RUN source activate cudf && conda install -c pytorch cython faiss-gpu
```

Comment:
Could we change this to install the cuda 9.2 version of faiss-gpu: `conda install faiss-gpu cuda92 -c pytorch` or perhaps make it user configurable to the correct cuda version?

Reply:
Added docker build-args for specifying cuda major and minor versions (easiest to work into the conda "cudaXY" package naming).